projects/plugins/jetpack/extensions/blocks/subscriptions/edit.js

```diff
@@ -199,7 +180,22 @@ export function SubscriptionEdit( props ) {
 	};
 
 	useEffect( () => {
-		getSubscriberCount();
+		getSubscriberCount(
+			count => {
+				setSubscriberCountString(
+					sprintf(
+						/* translators: Placeholder is a number of subscribers. */
+						_n( 'Join %s other subscriber', 'Join %s other subscribers', count, 'jetpack' ),
+						count
+					)
+				);
+				setSubscriberCount( count );
+			},
+			() => {
+				setSubscriberCountString( __( 'Subscriber count unavailable', 'jetpack' ) );
+				setSubscriberCount( 0 );
+			}
+		);
 	}, [] );
```

Comment:
Could we limit how often we make that request? Realistically, you may not need to do multiple times once you've opened the post editor.

Reply:
I confirmed that the subscribe block fires this `useEffect` three times upon initial insertion. However, it does seem to work as expected once per block addition for any additional subscribe blocks.

I agree - memoizing `getSubscriberCount` for a given sliding time window sounds like a good idea.